src/auth-service/controllers/join.js

```diff
@@ -137,7 +138,8 @@ const join = {
 
           res.status(status).json({
             success: false,
-            message: result.message,
+            message: "User does not exist",
+            exists: false,
```

Comment:
Return the message as "user does not exist" and the boolean value exists as false